miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -849,28 +852,35 @@ public static LibraryDto asDto(Library from, LibraryAdditionalInfo infoFrom) {
     }
     return dto;
   }
-  
-  public static Library to (LibraryDto from) {
-   Library to = new LibraryImpl();
-   to.setAlias(from.getAlias());
-   to.setName(from.getName());
-   to.setDescription(from.getDescription());
-   to.setInitialConcentration(from.getConcentration());
-   to.setLowQuality(from.getLowQuality());
-   to.setPlatformName(from.getPlatformName());
-   if (from.getTagBarcodeIndex1Id() != null) {
-     HashMap<Integer, TagBarcode> tagBarcodes = new HashMap<>();
-     TagBarcode tb1 = new TagBarcodeImpl();
-     tb1.setId(from.getTagBarcodeIndex1Id());
-     tagBarcodes.put(1, tb1);
-     if (from.getTagBarcodeIndex2Id() != null) {
-       TagBarcode tb2 = new TagBarcodeImpl();
-       tb2.setId(from.getTagBarcodeIndex2Id());
-       tagBarcodes.put(1, tb2);
-     }
-     to.setTagBarcodes(tagBarcodes);
-   }
-   to.setVolume(from.getVolume());
+
+  public static Library to(LibraryDto from) {
+    Library target = new LibraryImpl();
+    return to(from, target);
+  }
+
+  public static Library to(LibraryDto from, Library to) {
```

Comment:
This is not used and doesn't follow the pattern of other 'to' methods. Refactor into single 'to(LibraryDto from)' method unless there's a reason this is needed